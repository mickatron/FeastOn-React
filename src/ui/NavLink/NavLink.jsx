import { NavLink as ReactNavLink } from 'react-router-dom';
import React from 'react';

export default function NavLink(props) {
  return <ReactNavLink {...props} activeClassName="c-NavLink--active"/>;
}
