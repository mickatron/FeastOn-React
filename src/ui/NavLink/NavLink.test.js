
import { render } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import React from 'react';
import NavLink from './NavLink';

describe('NavLink', () => {

  let component = null;

  it('renders', () => {
    component = render(
      <MemoryRouter>
        <NavLink className="test" to="/test">label</NavLink>
      </MemoryRouter>
    );
    expect(component).toMatchSnapshot();
  });

});