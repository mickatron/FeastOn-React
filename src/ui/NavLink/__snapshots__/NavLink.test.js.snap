// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLink renders 1`] = `
<a
  aria-current="false"
  class="test"
  href="/test"
>
  label
</a>
`;
